src/executables/unload_schema.c

```diff
@@ -988,7 +1012,7 @@ emit_class_alter_serial (extract_context & ctxt, print_output & output_ctx)
 	      /* Validation of the result value */
```

Comment:
This switch is in the for-loop on line 1014.
`1014 for (i = 0; i < SERIAL_VALUE_INDEX_MAX; i++)`

Shouldn't this for-loop termination condition be changed ? (I'm not sure)
`1014 for (i = 0; i < ALTER_SERIAL_VALUE_INDEX_MAX; i++)`

Reply:
The array size of values ​​is missing.
Thanks for the advice.